pdns/slavecommunicator.cc

```diff
@@ -599,7 +599,8 @@ void CommunicatorClass::suck(const DNSName &domain, const ComboAddress& remote)
 
 
     g_log<<Logger::Error<<"AXFR done for '"<<domain<<"', zone committed with serial number "<<zs.soa_serial<<endl;
-    if(::arg().mustDo("slave-renotify"))
+    vector<string> meta;
+    if(::arg().mustDo("slave-renotify") || (B.getDomainMetadata(domain, "SLAVE-RENOTIFY", meta) && meta.size() > 0 && meta[0]=="1"))
```

Comment:
Please use the same logic as API-RECTIFY 

> When the API-RECTIFY value is “1”, the zone will be rectified on changes. Any other other value means that it will not be rectified. If this is not set at all, rectifying of the zone depends on the config variable default-api-rectify.`

SLAVE_RENOTIFY == 1 -> always renotify
SLAVE_RENOTIFY != 1 -> never renotify (this part is missing right now)

Reply:
Changed to similar logic